municipal_finance/settings.py

```diff
@@ -340,7 +339,7 @@
     "root": {"handlers": ["console"], "level": "ERROR"},
     "loggers": {
         "municipal_finance": {"level": "DEBUG" if DEBUG else "INFO"},
-        "sqlalchemy.engine": {"level": "INFO" if DEBUG else "WARN"},
+        # "sqlalchemy.engine": {"level": "INFO" if DEBUG else "WARN"},
```

Comment:
is this intentional or forgotten temporary comment?

Reply:
Guess I forgot it there. It speeds up a test run during development not to output all the queries.

Reply:
might be good to use a different variable than DEBUG so it's easier to tune log level, but not urgent right now